File: python/8 kyu/32.py
# https://www.codewars.com/kata/565f5825379664a26b00007c/train/python

# Passed

def get_size(w,h,d):
    area = 2 * (w * h + h * d + d * w)
    volume = w * h * d

    return [area, volume]

output = get_size(4, 8, 6)
print(output)

File: nodejs/7 kyu/32.js
// https://www.codewars.com/kata/52ed326b8df6540e06000029/train/javascript

// Passed

function goto(level, button) {
  // type cleansing
  if (typeof level !== "number") {
    return 0; // invalid level
  }

  if (typeof button !== "string") {
    return 0; // invalid button
  }

  button = Number(button);
  const validLevels = [0, 1, 2, 3];
  if (!validLevels.includes(level) | !validLevels.includes(button)) {
    return 0; // received invalid floor
  }

  return button - level;
}

const output = goto(1, '3');
console.log(output);